### ISSUE TITLE
Add Sort by Popularity

### ISSUE THREAD

**Issue by hal-shu-sato** [hal-shu-sato/apm#404]
## Is your feature request related to a problem? Please describe.

I thought sorting by popularity would make it easier to select plugins and scripts.
However, implementing this feature may be difficult because it requires user consent and a server to aggregate the data in order to collect usage data.

## Describe the solution you'd like

Add sort by popularity.

**Comment by yumetodo** [comment]
Sort by popularity sounds unfair as a package manager. There are no enough data set to measure it.

**Comment by hal-shu-sato** [comment]
That is true, but because of the current system, the order of display depends on the order in which the data is listed, and the order of display can change at the discretion of the maintainer.
I doubt that is a good thing.
I just put the idea down myself, but I think it needs to be considered whether to introduce it or not...

**Comment by yumetodo** [comment]
Instead of sorting by popularity, how about to filer by tag something like "apm developper selection"?

**Comment by mitosagi** [comment]
If we have the opportunity to post a video introducing apm, how about a survey at that time to determine the order of package recommendations? It would be good for publicity.

**Comment by hal-shu-sato** [comment]
Two are nice ideas!
At least, they are better than my idea.
Let's do them.
